docs/io/ascii/index.rst

```diff
@@ -99,8 +99,7 @@ disable the fast engine::
 .. Note::
 
    Reading a table which contains non-ASCII (unicode) characters is only
```

Comment:
This can be simplified to
```
Reading a table which contains unicode characters is supported; if you need
a different encoding, you can specify the ``encoding`` parameter in the
pure-Python readers.
```